### ISSUE TITLE
Unable to spawn WebWorker

### ISSUE THREAD

**Issue by jvilk** [ariya/phantomjs#13653]
When I try to run code that spawns a WebWorker, I receive the following error:

```
SECURITY_ERR: DOM Exception 18: An attempt was made to break through the security policy of the user agent.

  main.js:1
```

I am using PhantomJS 1.9.8, installed via npm. I tried running `main.js` in two ways, which both throw the same error:

```
$ phantomjs main.js
$ phantomjs --web-security=false main.js
```

**main.js:**

``` js
var worker = new Worker('worker.js');
worker.addEventListener('message', function (ev) {
    var dv = new DataView(ev.data);
});
```

**worker.js**:

``` js
while (1) {
    self.postMessage(new ArrayBuffer(0));
}
```

The problem seems identical to #10257, which was closed. Am I doing something wrong?

**Comment by zackw** [comment]
It looks like the fix for #10257 may have regressed.  This is definitely an issue even in 2.0.

I suspect the fix will be relatively easy but I don't have time to do it myself.

**Comment by jasmith79** [comment]
Still issue in 2.1.1.

**Comment by CodersBrothers** [comment]
The eternal error? Seems not fixed in 2.5.0
